### ISSUE TITLE
Full Accessibility & 508 Office Audit Find VA Forms Search

### ISSUE THREAD

**Issue by stephorkandatx** [department-of-veterans-affairs/va.gov-team#8694]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [x] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [x] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)

## Artifacts - _please provide the following artifacts when you request a full review_
- Access information for the tool or feature production:
  - https://va.gov/find-forms

### Required
[Discussed our situation in Slack re: having done staging review in prod.](https://dsva.slack.com/archives/C52CL1PKQ/p1594917633280800)

- [x] Conduct axe browser scan, and document or address issues found 
- [x] Ensure end-to-end tests - including the axe check, pass 
- [x] Check color pairing for proper contrast 
- [x] Check for colorblindness issues 
- [x] Zoom to 200% and confirm readability, no info cut off, etc 
- [x] Repeat for 300% 
- [x] Repeat for 400% 
- [x] Navigate all pages and core functionality with keyboard only 
- [x] Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox 

### Recommended

- [ ] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
- [ ] Add keyboard end-to-end tests in CI/CD test runs 
- [ ] Conduct axe-coconut browser scan, document or address issues found 
- [ ] Test with [WAVE](https://wave.webaim.org/) tool 

Known issues (if applicable):

* [#14432 508-defect-2 [COGNITION]: The page MUST specify the language](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14432)

* [#8248 components, design system 508-defect-2 [SCREENREADER] Aside MUST not be contained in another landmark; Main element should not enclose all content apart from header & footer](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8248)

* [#8250 CMS, components, design system, sitewide  508-defect-2 [AXE-CORE, SEMANTIC MARKUP]: Landmarks MUST have a unique role or role/label/title (i.e. accessible name) combination](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8250)

* [#8253 CMS, sitewide 508-defect-2 [SCREENREADER, KEYBOARD, COGNITION]: Links MUST look like links](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8253)

* [#8255 sitewide, design system, components 508-defect-3 [KEYBOARD]: Focused item MUST be contained by focus outline, pagination Next >](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8255)

* [#8279 508-defect-3 [FOCUS MANAGEMENT]: Focus outline SHOULD be removed from non-linked heading](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8279) 

## Additional Notes
As part of this review, VSA will send your product to VA's 508 office for approval (as applicable)

**Comment by stephorkandatx** [comment]
When the artifacts have been complete, let Jen Strickland know

**Comment by stephorkandatx** [comment]
Whoever picks this up - please sync with @jenstrickland before beginning to confirm the scope of what's needed + output for the checklist. I spoke with her and she thinks we can make this easier to get completed

**Comment by jenniferlee-dsva** [comment]
You can sync with me for now @jenstrickland.

**Comment by jenstrickland** [comment]
The current status of this ticket is that the team needs to complete the requirements above, to certify the above accessibility work is complete. 

To clarify, this does not require screenshots or an individual checklist for each page. Each step in the checklist is generally performed across the scope of the work being reviewed. Issues identified should be logged in issue tickets, and itemized as links under Known issues above. 

Once this is ready, teams generally comment here and then I will begin the full review, then send on to the VA 508 Office for their review.  Any questions, please let me know! 😃

**Comment by MickinSahni** [comment]
@jenstrickland @1Copenut made sure you're both assigned and checked off the items.

**Comment by jenstrickland** [comment]
Unassigning @1Copenut.

**Comment by jenstrickland** [comment]
Sent to VA 508 Program Office on Monday, November 9, 2020.

**Comment by MarciMcGuireGCIO** [comment]
@jenstrickland the last two items on this issue are closed.  Can this be closed out?  Thx!

**Comment by jenstrickland** [comment]
@MarciMcGuireGCIO We're still waiting on the report to come back from the VA 508 Office. These things can take months.

**Comment by MarciMcGuireGCIO** [comment]
Closed this by mistake

**Comment by joshkimux** [comment]
@MarciMcGuireGCIO 

@AngelaFowler82  and I finished our review of the new find a form features and identified no new defects from staging. This is the first full review we've completed with no new defects which is spectacular. Well done y'all, and extra credit to @zacharymorel for being such an a11y champ.  🥳 

@noahgelman for awareness!

**Comment by MarciMcGuireGCIO** [comment]
Whoo hoo!!!

**Comment by zacharymorel** [comment]
WHOO!
